ferrocene/doc/internal-procedures/src/upstream-pulls.rst

```diff
@@ -74,14 +88,67 @@ changed the file for is preserved. For example if we changed a bootstrap file,
 we need to port this change to another file. Usually this change needs to
 happen in the same PR.
 
-Some changes might be deferred to a follow-up PR. For example, if the deleted
-file was a test case used in the traceability matrix, we may
-need to find a new test case and finding it can be deferred.
+If the deleted file is a test and had annotations, it is necessary to go to the
+traceability-matrix and see whether the behavior is still covered by another test, 
+that is, that the tag is still listed in another test.
+
+As an example, upstream deleted the file `tests/ui/consts/miri_unleashed/mutable_references_err.rs`:
+
+.. code-block:: rust
+
+   <<<PULL-UPSTREAM>>> file deleted upstream; move the Ferrocene annotations if any, and delete this file
+  // ... 
+  // ferrocene-annotations: fls_qztk0bkju9u
+  // Borrow Expression
+  // ...
+
+Please make sure that the fls annotation still exists:
+
+.. figure:: figures/traceability-matrix.png
+
+   fls_qztk0bkju9u is still present in 83 tests
+
+If an annotation is lost, inform the team; to try to find another test that covers that behavior. 
+This can be done in a subsequent PR.
+
+
+Unknown conflicts and new errors
+"""""""""""""""""""""""""""""""" 
+
+When errors precendently unknown appear, such as a library not being found,
+try to isolate the faulty process that triggered it with `ferrocene/ci/split-tasks.py`. 
+The list of possible jobs can be found in `./circleci/workflow.yml`.
+
+Then check upstream for related fixes. If the failure was solved later, close the current PR
+and pull a new one, including the fix.
+
+Blessing
+""""""""
+
+Sometimes, the output of `stderr` has changed upstream.
+
+Run ``x test tests/<path-to-failing-tests-folder> --bless``
+
+Monitor CI
+^^^^^^^^^^
+
+If CI fail, it will fail early but not report. Do not hesitate to monitor CI, as bors can report 
```

Comment:
something like
```suggestion
If CI is failing, it will wait for the whole run before before reporting on GitHub. Do not hesitate to monitor CI, as bors can report 
```